flink-yarn-tests/src/test/java/org/apache/flink/yarn/testjob/YarnTestCacheJob.java

```diff
@@ -68,16 +73,21 @@ public static JobGraph getDistributedCacheJobGraph() {
 		@Override
 		public void open(Configuration config) throws IOException {
 			// access cached file via RuntimeContext and DistributedCache
-			File cacheFile = getRuntimeContext().getDistributedCache().getFile("cacheFile");
-			FileInputStream inputStream = new FileInputStream(cacheFile);
+			final File cacheFile = getRuntimeContext().getDistributedCache().getFile("cacheFile");
+			final FileInputStream inputStream = new FileInputStream(cacheFile);
 			properties = new Properties();
 			properties.load(inputStream);
 			checkArgument(properties.size() == 2, "The property file should contains 2 pair of key values");
+
+			final File testDirectory = getRuntimeContext().getDistributedCache().getFile("test_directory");
```

Comment:
Let's create a constant for `test_directory` so that we have a single place where the name is defined.

Reply:
Make sense. I will update.